packages/ui/src/app/pages/WorkingGroups/WorkingGroupsOpening.tsx

```diff
@@ -65,18 +65,14 @@ export const WorkingGroupOpening = () => {
   const rewardPeriod = useRewardPeriod(opening?.groupId)
 
   if (isLoading || !opening) {
-    return (
-      <PageLayout
-        lastBreadcrumb={id}
-        main={
-          <RowGapBlock gap={24}>
-            <ContentWithSidePanel>
-              <Loading />
-            </ContentWithSidePanel>
-          </RowGapBlock>
-        }
-      />
+    const Main = () => (
+      <RowGapBlock gap={24}>
+        <ContentWithSidePanel>
+          <Loading />
+        </ContentWithSidePanel>
+      </RowGapBlock>
     )
+    return <PageLayout lastBreadcrumb={id} main={Main} />
```

Comment:
What's the purpose of this? Without `useMemo` on `Main` I don't see a reason

Reply:
Reverted, even worse with useMemo
```
    const Main = useMemo(
      () => (
        (
          <RowGapBlock gap={24}>
            <ContentWithSidePanel>
              <Loading />
            </ContentWithSidePanel>
          </RowGapBlock>
        ),
        []
      )
    )
    return <PageLayout lastBreadcrumb={id} main={Main} />
```

Reply:
Why is it worse? Extracting `Main` to the new variable doesn't have any positive impact either on performance or code clarity. Going in-depth - this block of HTML is so small that extracting it to the new variable doesn't really improve code clarity, that's why I said that the only viable reason would be to keep it in `useMemo` to process it once and serve when needed, but in this case, this `if` block will be executed only once, so `useMemo` doesn't make much sense as well.

Summarizing there was no need to change this code at all

Reply:
worse as in more lines of code